packages/components/Organisms/Markup/Markup.vue

```diff
@@ -64,7 +64,7 @@ $callout-text-color: rpl_color('extra_dark_neutral') !default;
 $callout-author-ruleset: (rem(20px), 1.2em, 'medium') !default;
 $callout-mark-border: rem(4px) solid rpl_color('mid_neutral_2') !default;
 $callout-padding: 0 0 0 ($rpl-space * 6) !default;
-$callout-margin: ($rpl-space * 7) auto !default;
+$callout-margin: ($rpl-space * 7) 0 !default;
```

Comment:
@tim-yao this is the fix to markup callouts, if there is no side bar the content will auto align to center but the heading remains to the left. Please see below for the fix.
<img width="1284" alt="Screen Shot 2020-07-23 at 2 51 51 pm" src="https://user-images.githubusercontent.com/14160054/88253116-14b95c00-ccf4-11ea-9a4e-2743715ba4f5.png">